test/common/markdown/markdown-spell.tests.ps1

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Verify Markdown Spelling" {
+    BeforeAll {
+        if(!(Get-Command -Name 'mdspell' -ErrorAction SilentlyContinue)) {
+            Start-NativeExecution {
+                sudo yarn global add markdown-spellcheck@0.11.0
+            }
+        }
+
+        Get-Job | Remove-Job -Force
+    }
+
+    AfterAll {
+        Get-Job | Remove-Job -Force
+    }
+
+    $groups = Get-ChildItem -Path "$PSScriptRoot\..\..\..\*.md" -Recurse | 
```

Comment:
This script should be in the `BeforeAll` block -- Pester doesn't guarantee the semantics of things the occur in the body of the `Describe`